libraries/AP_HAL_SITL/AnalogIn.h

```diff
@@ -1,7 +1,9 @@
 #pragma once
 
 #include <AP_HAL/AP_HAL.h>
-#include "AP_HAL_SITL_Namespace.h"
+#if CONFIG_HAL_BOARD == HAL_BOARD_SITL
```

Comment:
files inside `libraries/AP_HAL_SITL` (or any other hal fwiw) doesn't need the guard because they are only compiled for that "board".